docs/data-sources/volume.md

```diff
@@ -140,6 +172,4 @@ Read-Only:
 
 - `description` (String) Description for the volume group.
 - `id` (String) Unique identifier of the volume group.
-- `name` (String) Name of the volume group.
```

Comment:
we also need to update the description if one of the name or id is a required parameter. Please check PowerFlex and update the descriptions at all places.